[bug] `go-aws-sso refresh` raises `panic: runtime error: invalid memory address or nil pointer dereference`

**Describe the bug**
Running `go-aws-sso refresh` does not work, even if `go-aws-sso` does.

**To Reproduce**
1. Run `go-aws-sso` and complete flow
2. Run `go-aws-sso refresh`

**Expected behavior**
After completing auth flow via web during `refresh`, auth is refreshed.

**Error Output**
```
❯ go-aws-sso refresh --debug
2024-07-02 12:46:01 DEBUG go-aws-sso/main.go:305 Debug logging enabled
2024-07-02 12:46:01 DEBUG go-aws-sso/main.go:238 Checking mandatory flags
2024-07-02 12:46:01 DEBUG sso/aws.go:275 No lock file found
2024-07-02 12:46:01 DEBUG sso/aws.go:115 Encountered error while reading client information: no ClientInformation exist
2024-07-02 12:46:02 WARN sso/aws.go:172 Please verify your client request: https://device.sso.us-east-1.amazonaws.com/?user_code=REDACTED
2024-07-02 12:46:02 INFO sso/aws.go:234 Still waiting for authorization...
2024-07-02 12:46:05 INFO sso/aws.go:234 Still waiting for authorization...
2024-07-02 12:46:08 INFO sso/aws.go:234 Still waiting for authorization...
2024-07-02 12:46:12 INFO sso/aws.go:234 Still waiting for authorization...
2024-07-02 12:46:15 INFO internal/refresh.go:31 Using Start URL https://REDACTED.awsapps.com/start/#
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x99e7c9]

goroutine 1 [running]:
github.com/theurichde/go-aws-sso/internal.RefreshCredentials({0xf58778, 0xc00007c3e0}, {0xf59438, 0xc00007c400}, 0xc0000ab240)
	/home/paarth/go/pkg/mod/github.com/theurichde/go-aws-sso@v1.4.4-0.20240701021535-bf3c7bed6075/internal/refresh.go:37 +0x369
main.main.func2(0xc0000ab240)
	/home/paarth/go/pkg/mod/github.com/theurichde/go-aws-sso@v1.4.4-0.20240701021535-bf3c7bed6075/cmd/go-aws-sso/main.go:103 +0x54
github.com/urfave/cli/v2.(*Command).Run(0xc000444420, 0xc0000ab240, {0xc0004590a0, 0x2, 0x2})
	/home/paarth/go/pkg/mod/github.com/urfave/cli/v2@v2.27.2/command.go:276 +0x97d
github.com/urfave/cli/v2.(*Command).Run(0xc000444840, 0xc0000aaec0, {0xc000024330, 0x3, 0x3})
	/home/paarth/go/pkg/mod/github.com/urfave/cli/v2@v2.27.2/command.go:269 +0xbb7
github.com/urfave/cli/v2.(*App).RunContext(0xc000446000, {0xf55508, 0x13e7e80}, {0xc000024330, 0x3, 0x3})
	/home/paarth/go/pkg/mod/github.com/urfave/cli/v2@v2.27.2/app.go:333 +0x58b
github.com/urfave/cli/v2.(*App).Run(...)
	/home/paarth/go/pkg/mod/github.com/urfave/cli/v2@v2.27.2/app.go:307
main.main()
	/home/paarth/go/pkg/mod/github.com/theurichde/go-aws-sso@v1.4.4-0.20240701021535-bf3c7bed6075/cmd/go-aws-sso/main.go:172 +0xe26
```

**Additional context**
Add any other context about the problem here.

--- a/internal/refresh.go
+++ b/internal/refresh.go
@@ -34,7 +34,6 @@ func RefreshCredentials(oidcClient ssooidciface.SSOOIDCAPI, ssoClient ssoiface.S
 	var roleName *string
 
 	lui, err := readUsageInformation()
-	zap.S().Infof("Attempting to refresh credentials for account [%s] with role [%s]", lui.AccountName, lui.Role)
 	if err != nil {
 		if strings.Contains(err.Error(), "no such file") {
 			zap.S().Info("Nothing to refresh yet")
@@ -50,6 +49,7 @@ func RefreshCredentials(oidcClient ssooidciface.SSOOIDCAPI, ssoClient ssoiface.S
 	} else {
 		accountId = &lui.AccountId
 		roleName = &lui.Role
+		zap.S().Infof("Attempting to refresh credentials for account [%s] with role [%s]", *accountId, *roleName)
 	}
 
 	rci := &sso.GetRoleCredentialsInput{AccountId: accountId, RoleName: roleName, AccessToken: &clientInformation.AccessToken}

